politeiad/cmd/politeia/README.md

```diff
@@ -28,6 +29,22 @@ Save to /home/user/.politeia/identity.json or ctrl-c to abort
 Identity saved to: /home/user/.politeia/identity.json
 ```
 
+## Verify a record
```

Comment:
Put this after the command that creates a new record and use the data from that command so it makes sense.

Reply:
done